src/__mocks__/context.ts

```diff
@@ -404,13 +402,14 @@ export class SyncIngestUpdateToPartInstanceContext extends RundownUserContext
 }
 
 // tslint:disable-next-line: max-classes-per-file
-export class ActionExecutionContext extends ShowStyleUserContext implements IActionExecutionContext {
+export class ActionExecutionContext extends ShowStyleUserContext implements ITV2ActionExecutionContext {
 	public currentPart: IBlueprintPartInstance
-	public currentPieceInstances: IBlueprintPieceInstance[]
+	public currentPieceInstances: Array<IBlueprintPieceInstance<PieceMetaData>>
 	public nextPart: IBlueprintPartInstance | undefined
-	public nextPieceInstances: IBlueprintPieceInstance[] | undefined
+	public nextPieceInstances: Array<IBlueprintPieceInstance<PieceMetaData>> | undefined
 
 	public takeAfterExecute: boolean = false
+	public isTV2Context: true = true
```

Comment:
I think either change the return type to boolean or delete this. If it can only be true then there is never a need to check for it :) 

Also, I don't seem to be able to spot where it is used, so I half assumed this is leftover from some testing? Or is there a hidden dependency I am missing?

Reply:
I agree :)